### DISCUSSION TITLE
PsPM for EDA during resting state and task-fMRI - SF models?

### DISCUSSION THREAD

**Discussion by PAmcconnell** [bachlab/PsPM#505] [Q&A] [answered]
Hi, I started corresponding with Dr. Bach via email and was directed here to post my questions about EDA analysis. I'm new to EDA data and working to find the best software solution to processing and modeling the data. I've been exploring PsPM and LedaLab and trying to understand the differences between the approaches. 

Briefly, we have skin conductance data collected over the course of four 6-min resting state fMRI runs. Here, I would like to look at spontaneous fluctuations (frequency, amplitude) and overall skin conductance levels.
 
We also have skin conductance collected over the course of six 8-min task fMRI runs using grip force during an implicit learning task. Each task block is separated into two blocks, sequence and random. I am interested to see if there are sympathetic nervous system metrics that distinguish the blocks and might predict learning outcomes.

I have been using the SF module but am having some difficulties interpreting the statistics (e.g., with output values < 1), especially in comparison to LedaLab. 
 
For the task-based analysis, my idea was to still use SF modeling since I don’t expect the activity of interest to be time-locked to the stimuli/individual task events. The sequence information is presented in a repeated pattern for half a run, and the random information for the other half run, with block order pseudo-randomized within and between subjects.
 
I’m looking to test if there are any general learning/encoding-related sympathetic indices that might differentiate the sequence and random blocks, or that predict learning of the sequence within and across runs (i.e., greater response precision). So, my thought was to define each block as an epoch in the SF model but also considering trying to model all six runs at once vs independently and running statistics on the output (e.g., by Group and Time).
 
We are collecting the EDA from the foot and I’m not noticing any major gradient artefacts (although not an expert).

(example data from one task run: 
[sub-LRN001_ses-2_task-learn_physio_run1_zeroed.mat.zip](https://github.com/bachlab/PsPM/files/12146230/sub-LRN001_ses-2_task-learn_physio_run1_zeroed.mat.zip)

Thanks in advance for any help and guidance on processing these data.

Best,
Patrick

**Comment by dominikbach** [comment, marked as answer]
This has been answered via email; I'll post the summary for sake of completeness.

You may want to use the SF module (with DCM or MP) for spontaneous fluctuations; you may want to make sure that your data contains no excessive gradient artefacts (in particular, downward spikes might be problematic with the modelling approach).

The stats output is the number of abobe-threshold spontaneous fluctuations in Hz. Since SF normally occur every few seconds, the values will usually be < 1. If you open the model output in matlab, you can also retrieve the amplitude and timing for each modelled response. Here, all timings are in "CNS" time rather than peripheral time, i.e. no need to take care of conduction delays. In turn, you benefit from including a few seconds of data after the epoch for inversion, and discarding any responses that were generated after the epoch ended.
